QemuPkg/Plugins/BuildSecurebootPcds/BuildSecurebootPcds.py

```diff
@@ -0,0 +1,80 @@
+# @file
+# A common helper script for any platform in the repository, that is used to generate
+# the necessary PCDs for secureboot.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+from edk2toolext.environment.plugintypes.uefi_helper_plugin import IUefiHelperPlugin
+import logging
+from pathlib import Path
+from edk2toollib.utility_functions import RunPythonScript
+import tempfile
+
+class BuildSecurebootPcds(IUefiHelperPlugin):
+    def RegisterHelpers(self, obj):
+        fp = str(Path(__file__).absolute())
+        obj.Register("generate_secureboot_pcds", BuildSecurebootPcds.generate_pcds, fp)
+        return 0
+    
+    @staticmethod
+    def generate_pcds(thebuilder):

```

Comment:
Since the return type is expected to be used by callers, I suggest at least adding a type hint for that.

```suggestion
    def generate_pcds(thebuilder) -> int:

```

Reply:
Updated. Thanks!